job_definitions/code_backups.yml

```diff
@@ -33,27 +33,25 @@
 {% for package in packages %}
 - job:
     name: 'backup-{{ package }}'
-    display-name: "Backup {{ package }} Github repo to AWS CodeCommit"
+    display-name: "[code-backup] {{ package }}"
```

Comment:
Could we use something other than `[`? This pushes the pipeline jobs down from the top of the main Jenkins list 😿

Reply:
Done 👍